src/test/scala/scala/concurrent/stm/FlipperSuite.scala

```diff
@@ -21,7 +21,7 @@ class FlipperSuite extends FunSuite {
   val DEFAULT_WORD_COUNT = 4096
   val DEFAULT_FLIP_PROB = 0.5f
   val DEFAULT_REF_ARRAY_FACTORY: Int => IndexedSeq[Ref[Int]] = { n =>
-    Array.tabulate[Ref[Int]](n) { _ => Ref(0) }
+    Array.tabulate[Ref[Int]](n) { _ => Ref(0) }.toIndexedSeq
```

Comment:
What about changing the return type to `collection.IndexedSeq` instead of calling `.toIndexedSeq`?

Reply:
They want immutable data.